config/application.rb

```diff
@@ -1,35 +1,43 @@
-require File.expand_path('../boot', __FILE__)
+require_relative 'boot'
 
 require "rails"
 # Pick the frameworks you want:
 require "active_model/railtie"
 require "active_job/railtie"
 require "active_record/railtie"
+require "active_storage/engine"
 require "action_controller/railtie"
 require "action_mailer/railtie"
 require "action_view/railtie"
+# require "action_cable/engine"
```

Comment:
What's this?

Reply:
[WebSockets Support](https://guides.rubyonrails.org/action_cable_overview.html)! Pretty nifty, but not needed for this app imo